filebeat/module/elasticsearch/gc/_meta/fields.yml

```diff
@@ -1,20 +1,110 @@
 - name: gc
   type: group
   description: >
+    GC fileset fields.
   fields:
-    - name: relative_process_timestamp_secs
+    - name: phase
+      type: group
+      description: >
+        Fields specific to GC phase.
+      fields:
+        - name: name
+          type: keyword
+          description: >
+            Name of the GC collection phase.
+        - name: duration_sec
```

Comment:
We should follow the naming convention here: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html

You can add here also `format: duration` so Kibana will not it's a duration.